### PR TITLE
Convert HTML to use reSpec references

### PR DESCRIPTION
**Pull request by afs**
This is material kindly provided from @gkellogg to switch from HTML references to the way reSpec manages references. 

There are also HTML fixes.

There are no content changes.

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Feb 5, 2023, 4:51 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fsparql-query%2F00d5ee93e21b5bd473ffb781f1901b2488c0b2bc%2Fspec%2Findex.html%3FisPreview%3Dtrue)

```

😭  Sorry, there was an error generating the HTML. Please report this issue!
Specification: http://labs.w3.org/spec-generator/uploads/TnA9dQ/spec/index.html?isPreview=true%3FisPreview%3Dtrue&publishDate=2023-02-05
ReSpec version: 32.7.0
File a bug: https://github.com/w3c/respec/
Error: Error: Evaluation failed: Timeout: document.respec.ready didn't resolve in 27666ms.
    at ExecutionContext._ExecutionContext_evaluate (/u/spec-generator/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:229:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ExecutionContext.evaluate (/u/spec-generator/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:107:16)
    at async generateHTML (/u/spec-generator/node_modules/respec/tools/respecDocWriter.js:221:12)
    at async toHTML (/u/spec-generator/node_modules/respec/tools/respecDocWriter.js:92:18)
    at async Object.generate [as respec] (file:///u/spec-generator/generators/respec.js:15:44)
    at async file:///u/spec-generator/server.js:252:48

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/sparql-query%234.)._
</details>

### CODE REVIEW

**Comment by afs** [comment]
Current work-in-progress.

cc @kasei  @hartig  @rubensworks

File: spec/index.html [hunk 1]
```diff
@@ -105,189 +110,275 @@ <h2>Abstract</h2>
...
+      <p>
+        RDF is a directed, labeled graph data format for representing information in the Web. RDF is
+        often used to represent, among other things, personal information, social networks, metadata
+        about digital artifacts, as well as to provide a means of integration over disparate sources of
+        information. This specification defines the syntax and semantics of the SPARQL query language
+        for RDF.
+      </p>
+      <p>
+        The SPARQL query language for RDF is designed to meet the use cases and requirements
+        identified by the RDF Data Access Working Group in [[[?UCNR]]] [[?UCNR]] and [[[?UCNR2]]]
```
Comment (gkellogg) [inline spec/index.html:127]: Note that ReSpec typically figures out if references are normative or informative depending on if they're in a normative/informative section. You can, of course, override this using `[[?UCNR]]` or `[[!UCNR]]`. Not wrong to do it, but just FYI.

File: spec/index.html [hunk 2]
```diff
@@ -105,189 +110,275 @@ <h2>Abstract</h2>
...
+          an RDF term. Variables are not required to be bound in a solution.</p>
+      </section>
+      <section id="docTerminology">
+        <h4>Terminology</h4>
+        <p>The SPARQL language includes IRIs, a subset of RDF URI References that omits spaces.
+          Note that all IRIs in SPARQL queries are absolute; they may or may not include a fragment
+          identifier [[RFC3987], section 3.1. IRIs include URIs [[RFC3986]] and URLs. The abbreviated
+          forms (<a href="#QSynIRI">relative IRIs and prefixed names</a>) in the SPARQL syntax are
+          resolved to produce absolute IRIs.</p>
+        <p>The following terms are defined in [[[RDF11-CONCEPTS]]] [[RDF11-CONCEPTS]] and used in
```
Comment (gkellogg) [inline spec/index.html:298]: Switch to `RDF12-CONCEPTS`?

File: spec/index.html (hunk 1 above)
Comment (afs) [reply spec/index.html:127]: The `?` came from your PR. i was wondering what it did!

File: spec/index.html (hunk 2 above)
Comment (afs) [reply spec/index.html:298]: `s/RDF11-/RDF12-/g`

File: spec/index.html (hunk 1 above)
Comment (gkellogg) [reply spec/index.html]: I think the original reference was explicitly informative, so I probably just replicated that. It’s not worth scouring the spec to save a couple of characters, unless informative references should be normative or visa-versa.

**Pull request merged**